src/game/scenes/Game.js

```diff
@@ -4,20 +4,61 @@ import AnimatronicsNames from '../utils/AnimatronicsNames';
 
 import Animatronic from '../entities/Animatronic';
 
+import Cup from '../entities/Cup';
+
 export default class Game extends Phaser.Scene {
   constructor() {
     super('Game');
   }
 
   async create() {
-    this.matter.world.setBounds(0, 0, this.game.config.width, this.game.config.height, 32, true, true, false, true);
+    this.matter.world.setBounds(
+      0,
+      0,
+      this.game.config.width,
+      this.game.config.height,
+      32,
+      true,
+      true,
+      false,
+      true,
+    );
 
     const centerX = this.scale.width / 2;
     const centerY = this.scale.height / 2;
 
-    const customObject = new Animatronic(this, AnimatronicsNames.BB, centerX, centerY);
+    const customObject = new Animatronic(
+      this,
+      AnimatronicsNames.BB,
+      centerX,
+      centerY,
+    );
+    const cupPart1 = new Cup(
```

Comment:
I think that the Cup should be a single object that we can add wherever we want on the scene.